clusters/app.ci/release-controller/deploy-ocp-arm64-controller.yaml

```diff
@@ -238,6 +241,10 @@ spec:
           defaultMode: 420
           name: job-config-4.10
         name: job-config-410
+      - configMap:
```

Comment:
nit: would be nicer with `project`. (I knew this is controlled by a python script.)

Reply:
Yeah, unfortunately these bits are generated - so at least it's not hard to maintain them.

Reply:
I'm more than happy to take a look at this if it's something that can be improved.  Im just not sure I understand what you mean by `would be nicer with project`?

Reply:
We had to move to auto-generating these pieces because mistakes, while manually editing these files, were killing us.  Certainly willing to make any enhancements to make the whole process easier!

Reply:
I think Hongkai means `projected` volumes which allow you to project multiple sources into a single volume and you can then mount it at once. We did it in https://github.com/openshift/release/pull/24593/commits/ce98548fe4577e3ece00a3cef666fb4ee7e86612

Reply:
Gotcha!  Thanks for the clarification.

Reply:
I think this should do what you've suggested:
https://github.com/openshift/release/pull/24665